posts/2025-02-17_theres_a_pharmaverse_package_for_that/managing-clinical-trial-data.qmd

```diff
@@ -0,0 +1,323 @@
+
+    -   [**Tplyr**](https://github.com/atorus-research/Tplyr) for clinical summary tables
+
+    -   The pharmaverse offers multiple table-making packages, including [rtables](https://insightsengineering.github.io/rtables/v0.6.10/articles/rtables.html), [chevron](https://insightsengineering.github.io/chevron/latest-tag/), [pharmaRTF](https://atorus-research.github.io/pharmaRTF/), [gtsummary](https://www.danieldsjoberg.com/gtsummary/), [cards](https://insightsengineering.github.io/cards/latest-tag/), [tfrmt](https://gsk-biostatistics.github.io/tfrmt/), [tidytlg](https://pharmaverse.github.io/tidytlg/main/), and [cardinal](https://pharmaverse.github.io/cardinal/). You can find more tools on the [TLGs page](https://pharmaverse.org/e2eclinical/tlg/). Each serves different needs, and it is up to the programmer to choose the best fit.
+
+5.  **Submission** – Final reporting and submission:
+
+    -   [**xportr**](https://github.com/atorus-research/xportr) for CDISC-compliant dataset export.
+
+    -   [**pkglite**](https://merck.github.io/pkglite/) for package management and tracking.
+
+By using these tools, organizations can optimize their data pipeline, ensuring clinical data is well-structured, validated, and ready for regulatory submission with ease.
+
+## **Where Do I Start? Understanding Tasks & Objectives**
+
+For those new to clinical reporting in R, understanding the foundational tasks and objectives is critical.
+
+#### **Metadata-Driven Clinical Reporting Approach**
+
+A metadata-driven method ensures clinical trial data is consistently structured and aligned with regulatory standards. The process typically follows this sequence:
+
+[**Metadata**](https://pharmaverse.org/e2eclinical/metadata/) **\> [OAK](https://github.com/pharmaverse/sdtm.oak) \> [Admiral](https://github.com/pharmaverse/admiral) \> Define.xml \> [TLGs](https://pharmaverse.org/e2eclinical/tlg/) (rtf/pdf) \> Submissions \> Shiny**
+
+This structured approach makes data preparation for regulatory submission more transparent and traceable.
+
+#### **Essential Resources & Packages**
+
+To effectively implement this process, organizations should become familiar with essential packages:
+
+-   **Metadata Management:** [metacore](https://github.com/atorus-research/metacore), [metatools](https://pharmaverse.github.io/metatools/) – for standardized metadata structures and validation.
+
+-   [**SDTM (Standard Data Tabulation Model)**](https://pharmaverse.org/e2eclinical/sdtm/)**:** Ensures clinical data follows CDISC standards.
+
+-   **Validation Tools:** [sdtmchecks](https://github.com/pharmaverse/sdtmchecks), [datacutr](https://cran.r-project.org/web/packages/datacutr/index.html) (ensuring data integrity and regulatory compliance).
+
+For more details, explore:
+
+-   **Clinical Metadata Resources:** <https://pharmaverse.org/e2eclinical/metadata>
+
+-   **SDTM Standards & Tools:** <https://pharmaverse.org/e2eclinical/sdtm> 
+
+## **Example: Creating ADSL**
```

Comment:
Should we just advertise the examples site here and https://pharmaverse.github.io/examples/adam/adsl.html rather than repeating this within the blog?